geoips/plugins/yaml/sectors/static/denver.yaml

```diff
@@ -20,9 +20,9 @@ spec:
     proj: eqc
     units: m
   resolution:
-    - 100
-    - 100
+    - 500
+    - 500
   shape:
-    height: 2000
-    width: 2000
+    height: 400
+    width: 400
```

Comment:
Are we ok with this being our sample Denver sector?

Reply:
Yeah, this is a little weird. A 2000x2000 km image for a city?

I just updated to be 250x250m resolution and 200x200 shape.